wallet/src/de/schildbach/wallet/ui/invite/InviteDetailsFragment.kt

```diff
@@ -102,16 +102,21 @@ class InviteDetailsFragment : InvitationFragment(R.layout.fragment_invite_detail
             }
         }
         profile_button.setOnClickListener {
-            viewModel.invitedUserProfile.observeOnce(requireActivity(), Observer {
-                if(it != null) {
-                    startActivity(DashPayUserActivity.createIntent(requireContext(), it))
-                } else {
-                    /*not sure why this is happening*/
-                    val errorDialog = FancyAlertDialog.newProgress(R.string.invitation_creating_error_message_not_synced,
-                            R.string.invitation_verifying_progress_title)
-                    errorDialog.show(childFragmentManager, null)
-                }
-            })
+            viewModel.invitedUserProfile.observeOnce(
```

Comment:
I think I had removed observeOnce since it can be fully replaced with suspend functions.
I can merge `dashpay` into `feature-dashpay-coinjoin` later to resolve the conflicts.